channel/consensus_channel/leader_channel_test.go

```diff
@@ -207,10 +206,9 @@ func TestLeaderChannel(t *testing.T) {
 			guarantee(vAmount, channel1Id, alice, bob),
 		)
 		c := testChannel(startingOutcome, emptyQueue())
-		proposalMade := createAdd(cId, 1, targetChannel)
-		expectedSp := aliceSignedProposal(c.current.Vars, proposalMade).SignedProposal
+		proposalMade := createAdd(cId, targetChannel)
+		expectedSp := aliceSignedProposal(c.current.Vars, proposalMade, 1).SignedProposal
 
-		proposalMade.SetTurnNum(9001)
```

Comment:
This test case can be removed now? the block

```
{
		msg := "ok:provided turn number is ignored"
		startingOutcome := makeOutcome(
			allocation(alice, aBal-amountAdded),
			allocation(bob, bBal),
			guarantee(vAmount, channel1Id, alice, bob),
		)
		c := testChannel(startingOutcome, emptyQueue())
		proposalMade := createAdd(cId, targetChannel)
		expectedSp := aliceSignedProposal(c.current.Vars, proposalMade, 1).SignedProposal

		t.Run(msg, testPropose(c, proposalMade, expectedSp, nil))
	}
```

Reply:
Good catch! Removed in https://github.com/statechannels/go-nitro/pull/620/commits/cf1888ae7f85e54734118c6fcbe0edccd423ebac